src/main/java/frc/robot/swervev2/SwervePosEstimator.java

```diff
@@ -58,44 +76,132 @@ public SwervePosEstimator(GenericEncodedSwerve frontLeftMotor, GenericEncodedSwe
                 new Pose2d(),
                 stateStdDevs,
                 visionMeasurementStdDevs);
+        estimatedPose = new AtomicReference<>(poseEstimator.getEstimatedPosition());
         NetworkTableInstance inst = NetworkTableInstance.getDefault();
         NetworkTable table = inst.getTable("ROS");
-        subscriber = table.getDoubleArrayTopic("Pos").subscribe(new double[]{-1,-1,-1});
+        visionMeasurementSubscriber = table.getDoubleArrayTopic("Pos").subscribe(new double[]{-1,-1,-1,-1}, PubSubOption.pollStorage(10), PubSubOption.sendAll(true));
```

Comment:
sendAll with queue 20 messages, there is no reason that we would ever have more than 10. In reality, it does not matter because we read the messages quick enough that it won't even get to 10. I am not sure about the functionality of just having pollStorage we would have to test it. (When I coded this I did not know how many vision measurement we were getting in one tick